packages/next/next-server/server/next-server.ts

```diff
@@ -549,6 +551,12 @@ export default class Server {
           }
         },
       },
+      {
+        match: route('/_next/image'),
+        type: 'route',
+        name: '_next/image catchall',
+        fn: (req, res) => imageOptimizer(server, req, res),
```

Comment:
```suggestion
        fn: (req, res, _params, parsedUrl) => imageOptimizer(server, req, res, parsedUrl),
```
We can use the `parsedUrl` param here so that we don't have to re-parse it to get the query values